python-sdk/python_sdk_custom_docker_demo/server.py

```diff
@@ -0,0 +1,17 @@
+import sys
+import time
+import os
+from datetime import datetime
+
+#this show you how you can get input
+input = open(sys.argv[1]).read()
+
+#this show if you use a python library
+now = datetime.now()
+current_time = now.strftime("%H:%M:%S")
+print("Current Time =", current_time)
+
+#this show you how can you run a custom library script
+os.system("wget -P ./run-result " + input)
```

Comment:
subprocess is the preferred way if you want to educate the user

Reply:
:D I forgot how to use subprocess properly. I will leave that for now?

Reply:
https://docs.python.org/3.7/library/subprocess.html